packages/components/Notification/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export { Notification } from './Notification';
-export type { NotificationTokens, NotificationProps, NotificationSlotProps, NotificationType } from './Notification.types';
+export * from './Notification.types';
```

Comment:
I think we prefer not Tod o export * because of some Javascript bundler optimization thing. Could we actually go back to explicitly exporting the things?